hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/utils/db/Table.java

```diff
@@ -396,13 +392,8 @@ public V getValue() {
       }
 
       @Override
-      public byte[] getRawKey() throws IOException {
-        return rawKey;
-      }
-
-      @Override
-      public byte[] getRawValue() throws IOException {
-        return rawValue;
+      public int getReadableBytes() throws IOException {
+        return readableBytes;
```

Comment:
nit: I think `readableBytes` is in `CodecBuffer` terms (which supports reading the data from the buffer).  For `KeyValue`, it may be better to rename this to `rawSize`.